lib/Twig/Extensions/Node/Trans.php

```diff
@@ -41,6 +41,10 @@ public function compile(Twig_Compiler $compiler)
 
         $function = null === $this->getNode('plural') ? 'gettext' : 'ngettext';
 
+        if (null !== $this->getNode('domain')) {
```

Comment:
Ah, no, it is not that easy. What if we say if domain is set, then a string value must be passed?